### DISCUSSION TITLE
使用版面恢复官方文档中的图片(截取左边部分)，表格没有被还原为表格

### DISCUSSION THREAD

**Discussion by xiejingchao** [PaddlePaddle/PaddleOCR#13758] [Q&A]
### 🔎 Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/docs/recovery/recovery_ch.jpg
使用版面恢复官方文档中的图片(截取左边部分)，表格没有被还原为表格而是整张图片。

### 🏃‍♂️ Environment (运行环境)

debian 10
python 3.10
paddlepaddle 2.5.2
paddleocr 2.6.1

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

paddleocr --image_dir=./demo.png --type=structure --recovery=true

**Comment by xiejingchao** [comment]
![demo](https://github.com/user-attachments/assets/f0c6bd9e-8258-4db7-9997-09c1bcb8630b)
就是这张截图，识别后下面是图片不是表格
![11](https://github.com/user-attachments/assets/c31890f0-dcde-4016-aea5-dae4517a00d9)

**Comment by UserWangZz** [comment]
![image](https://github.com/user-attachments/assets/61e7d85f-67b1-4404-a756-237035400d92)
你好，我这边运行正常，paddleocr=2.8.1，paddle=2.6.0，尝试重新运行一下程序或者升级一下paddleocr版本

**Reply by xiejingchao** [reply]
你的原图怎么和我的不一样，貌似因为你的图表格左右有更宽的边距。
